packages/react/src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -8,6 +8,8 @@ import React, { useEffect, useLayoutEffect, useRef } from 'react';
 import Button from '../../internal/vendor/carbon-components-react/components/Button/Button';
 import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings';
 import PropTypes from 'prop-types';
+import root from 'window-or-global';
+import { sameHeight } from '@carbon/ibmdotcom-utilities';
```

Comment:
I believe we are now importing exports like so

`import  sameHeight  from '@carbon/ibmdotcom-utilities/es/utilities/sameHeight/sameHeight';`

to optimize bundle size